tests/test-common.cc

```diff
@@ -196,6 +196,7 @@ TEST_CASE("cpu_percentage and cpu_barval return correct values") {
     REQUIRE(cpu_barval(&obj1) == Approx(0.507));
 
     delete[] info.cpu_usage;
+    info.cpu_usage = nullptr;
```

Comment:
This little thing is out of scope, but it causes any tests which attempt using `CRIT_ERR`, that happen after test-common (i.e. all of them) to crash. Tests shouldn't use `CRIT_ERR` because it interrupts their execution, but it's better to handle it explicitly than as a side effect of another test.